migrations/sql/V11_AddLakeSorrento.sql

```diff
@@ -0,0 +1,3 @@
+insert into lake (id, name, supports_temperature, supports_tides)
+values ('4ddb043b-d0d2-44a5-b321-6efbaacf98a0', 'Tyrrhenisches Meer (Sorrento)', true, false)
```

Comment:
1. The Tyrrhenian Sea is part of the Mediterranean Sea. We already phoned it in for Heraklion so let's do it here as well for consistency
2. The German name for Sorrento is Sorrent

```suggestion
values ('4ddb043b-d0d2-44a5-b321-6efbaacf98a0', 'Mittelmeer (Sorrent)', true, false)
```